client/test/unit/specs/transactions.spec.js

```diff
@@ -356,10 +356,10 @@ describe('Testing Transactions.vue EXPIRED', () => {
     expect(vm.$el.querySelector('#transaction-header-names').textContent).toContain('Not good')
 
     expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Submitted Date:')
-    expect(vm.$el.querySelector('.submitted-date').textContent).toBe('2021-07-12, 9:49 am Pacific time')
+    expect(vm.$el.querySelector('.submitted-date').textContent).toBe('2022-01-13, 9:49 am Pacific time')
 
     expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Expiry Date:')
-    expect(vm.$el.querySelector('.expiry-date').textContent).toBe('2021-09-01, 12:01 am Pacific time')
+    expect(vm.$el.querySelector('.expiry-date').textContent).toBe('2022-03-16, 11:59 am Pacific time')
```

Comment:
Again, 11:59 am is incorrect. Should be pm.

Reply:
will fix it

Reply:
done.